Formula/s/scoutsuite.rb

```diff
@@ -0,0 +1,553 @@
+class Scoutsuite < Formula
+  include Language::Python::Virtualenv
+
+  desc "Open source multi-cloud security-auditing tool"
+  homepage "https://github.com/nccgroup/ScoutSuite"
+  url "https://files.pythonhosted.org/packages/1f/9c/3d8b7323ef163f1fe3f4d1176676ba474e4bd2568cfd6004d6f9ebcd05a3/ScoutSuite-5.13.0.tar.gz"
+  sha256 "0ad1290c274324b18d44095d4d8a5c5d5fa71c9474ed65a3990e942f3bcbf281"
+  license "GPL-2.0-only"
+  head "https://github.com/nccgroup/ScoutSuite.git", branch: "master"
+
+  #
```

Comment:
```suggestion

```

Reply:
Also had to reduce the line length but fixed now :)